inputFiles/poromechanics/ThermoElastic_2d_benchmark.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Included>
+    <File name="./ThermoElastic_2d_base.xml" />

```

Comment:
Should a smoke test be added to the integratedTest for this specific example? Or a separate PR will handle it?

Reply:
Yes you right, I changed the name to `ThermoElastic_2d_smoke.xml` because this really small test case was meant to be a smoke test